class-wp-bootstrap-navwalker.php

```diff
@@ -252,7 +252,7 @@ public function start_el( &$output, $item, $depth = 0, $args = array(), $id = 0
 			 *
 			 * Not having escape_titles set, or setting escape_titles to true will result in escaped titles.
 			 */
-			if (property_exists($args, 'escape_titles') && false === $args->escape_titles) {
+			if (property_exists($args, 'escape_titles') && false === $args->escape_titles) { // WPCS: loose comparison ok.
```

Comment:
No need to flag this for ignore since you updated it to use strict (`===`) comparisons :)

Reply:
Awesome!